### PR TITLE
Add use cases overview

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: astro/src/content/docs/get-started/use-cases/_api-consents-platform-description.mdx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+If you APIs that others are building on top of, use FusionAuth to manage user data, and want to allow your users to delegate access to their data accessible via your APIs, FusionAuth can handle the full OAuth grant, including custom scopes and customization of access tokens.
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/use-cases/_api-consents-platform-description.mdx:1]:
```suggestion
If you have APIs that others are building on top of, use FusionAuth to manage user data, and want to allow your users to delegate access to their data accessible via your APIs, FusionAuth can handle the full OAuth grant, including custom scopes and customization of access tokens.
```

File: astro/src/content/docs/get-started/use-cases/_authorization-hub-description.mdx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Safely manage your user's social access tokens in one place. Add new social providers as needed. Secure the refresh tokens and other credentials for social providers like Google, Instagram, YouTube, Facebook, and more.
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/use-cases/_authorization-hub-description.mdx:1]: Should this be plural possessive  users' ?

File: astro/src/content/docs/get-started/use-cases/_m2m-communication-description.mdx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Manage authentication and permissions for software programs using the client credentials grant and access tokens. More secure than static API keys, with fine grained permissions and service identity built in. 
+
+Self-host FusionAuth and embed it into your Kubernetes cluster, on-prem data center or other environment. 
+
+FusionAuth can scale to hundreds of millions of entities (APIs, devices) and you can manage the entire solution via language specific SDKs.
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/use-cases/_m2m-communication-description.mdx:5]: may want to link to blog you did on this with the performance.

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
one change a few comments

File: astro/src/content/docs/get-started/use-cases/_m2m-communication-description.mdx (hunk 3 above)
Comment (mooreds) [reply astro/src/content/docs/get-started/use-cases/_m2m-communication-description.mdx:5]: thanks! I don't typically link to blogs from docs (other way around is more common) so I think I'll leave this as is. Appreciate the suggestion.

**Review by mark-robustelli — APPROVED** [review]
Good to go.

**Pull request merged**